auto-configurations/vector-stores/spring-ai-autoconfigure-vector-store-opensearch/src/main/java/org/springframework/ai/vectorstore/opensearch/autoconfigure/OpenSearchVectorStoreProperties.java

```diff
@@ -144,7 +144,7 @@ public void setAws(Aws aws) {
 		this.aws = aws;
 	}
 
-	static class Aws {
```

Comment:
We provide public accessor/setter for this although this is a package-protected inner class. Since we can't change those public API's, we should make this inner `AWS` class as public.